dev-packages/e2e-tests/package.json

```diff
@@ -26,7 +26,7 @@
     "yaml": "2.2.2"
   },
   "volta": {
-    "node": "18.17.1",
```

Comment:
Why the bump?

Reply:
Codecov plugins require min Node version of `18.18.0`.